backend/src/main/kotlin/app/ehrenamtskarte/backend/auth/database/repos/ApiTokensRepository.kt

```diff
@@ -18,4 +19,8 @@ object ApiTokensRepository {
             this.projectId = projectId
         }
     }
+
+    fun findByToken(token: ByteArray): ApiTokenEntity? {
+        return ApiTokenEntity.find { (ApiTokens.token eq token) }.singleOrNull()
```

Comment:
Probably, we should add an index for the token column of ApiTokens then?
And maybe we should rename the column to `tokenHash`?